modules/button/react/lib/Button.tsx

```diff
@@ -69,27 +76,35 @@ export default class Button extends React.Component<ButtonProps> {
   }
 }
 
+/**
+ * @deprecated This is now deprecated, use Button instead
```

Comment:
We will have to add a writeup about the deprecation in the readme. Let's link to that from all of the `@deprecated` tags.